### ISSUE TITLE
Longer default retry period

### ISSUE THREAD

**Issue by szpak** [Codearte/gradle-nexus-staging-plugin#12]
Find out sensible algorithm.

**Comment by adamdubiel** [comment]
How can i check retries count? Should i build my project with `-i`?

**Comment by szpak** [comment]
Yes, `-i` is enough. I will add it to a README file.

**Comment by adamdubiel** [comment]
Okay, i will add this to our release plan and get back to you when i have some data.

**Comment by szpak** [comment]
Great, thanks.

**Comment by adamdubiel** [comment]
Getting back to this issue, i have some data from our new [hermes](https://github.com/hermes) release:

```
:closeRepository (Thread[Daemon worker,5,main]) started.
build   07-Aug-2015 11:27:38    :closeRepository
build   07-Aug-2015 11:27:38    Executing task ':closeRepository' (up-to-date check took 0.0 secs) due to:
build   07-Aug-2015 11:27:38      Task has not declared any outputs.
build   07-Aug-2015 11:27:39    Getting staging profile for package group pl.allegro
build   07-Aug-2015 11:27:42    Getting open repository for staging profile 19d6feb4b1fb3
build   07-Aug-2015 11:27:42    Closing repository plallegro-1075 with staging profile 19d6feb4b1fb3
build   07-Aug-2015 11:27:43    POST response data: null
build   07-Aug-2015 11:27:43    Repository plallegro-1075 with staging profile 19d6feb4b1fb3 has been closed
build   07-Aug-2015 11:27:43    :closeRepository (Thread[Daemon worker,5,main]) completed. Took 4.802 secs.
build   07-Aug-2015 11:27:43    :promoteRepository (Thread[Daemon worker,5,main]) started.
build   07-Aug-2015 11:27:43    :promoteRepository
build   07-Aug-2015 11:27:43    Executing task ':promoteRepository' (up-to-date check took 0.0 secs) due to:
build   07-Aug-2015 11:27:43      Task has not declared any outputs.
build   07-Aug-2015 11:27:43    Using configured staging profile id: 19d6feb4b1fb3
build   07-Aug-2015 11:27:43    Getting closed repository for staging profile 19d6feb4b1fb3
build   07-Aug-2015 11:27:44    Attempt 1/16 failed. IllegalArgumentException was thrown with message 'Unexpected state of reveived repository. Received open, expected closed'. Waiting 5000 ms before next retry.
build   07-Aug-2015 11:27:49    Getting closed repository for staging profile 19d6feb4b1fb3
build   07-Aug-2015 11:27:50    Promoting repository plallegro-1075 with staging profile 19d6feb4b1fb3
build   07-Aug-2015 11:27:51    POST response data: null
build   07-Aug-2015 11:27:51    Repository plallegro-1075 with staging profile 19d6feb4b1fb3 has been promotted
build   07-Aug-2015 11:27:51    :promoteRepository (Thread[Daemon worker,5,main]) completed. Took 7.521 secs.
```

**Comment by szpak** [comment]
I increased default retry count to 20 and delay between retries to 2000 - ~40 seconds + request/response time in 0.7.0. Should be enough in most cases. Feel free to open it if the new default values are still too low and dedicated algorithm for multimodule projects is really needed.
